src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -0,0 +1,79 @@
+/* global cozy */
+import React from 'react'
+import FuzzyPathSearch from '../FuzzyPathSearch'
+
+class SuggestionProvider extends React.Component {
```

Comment:
Without a render method, I don't see the point of inheriting from React.Component here...

Reply:
I think it's necessary because we're using other parts of the lifecycle, specifically `componentDidMount`. So I think a regular class wouldn't work, and a functional component will be called too often. But maybe I'm missing something?

Reply:
I think the code in the `componentDidMount` could be put in a regular constructor of a regular class, but maybe it's me that's missing something :p

Reply:
It could, but then you can't use that class as a component [here](https://github.com/cozy/cozy-drive/pull/394/files#diff-ccfb11fef287f9616f7aa1a57b1209a5R55) 🤷‍♂️